### PR TITLE
Drawer and bio section added:Mobile Version 

### PR DESCRIPTION
**Pull request by habitoz**
added html and css files with the following changes
    - header added with navigation links and drawer icon.
    - bio section added with title and bio info inside it.
    - style.css file created with all style rules from the design (given template)

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @habitoz, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that kindly add linter to your project so that it would guide you on best practices.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            </p>
+            <div class="link">
+                <p class="link-text">LET'S CONNECT</p>
+                <div class="profile-links">
+                    <a href=""><img src="./assets/images/twitter.svg" alt="" srcset=""></a>
+                    <a href=""><img src="./assets/images/m.svg" alt="" srcset=""></a>
+                    <a href="https://github.com/habitoz"><img src="./assets/images/github.svg" target="_blank"  alt="" srcset=""></a>
+                    <a href=""><img src="./assets/images/ok.svg" alt="" srcset=""></a>
+                </div>
+            </div>
```
Comment (iambenkis) [inline index.html]: - [ ] Instaed of using a `div` that has `a` tag , kindly use `ul`  and `li` to represent this list of social media icons

```bash
<ul>
      <li></li>
       ...
</ul>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <title>Portofolio</title>
+</head>
+<body>
+    <header class="header">
+      <div class="logo">My Logo</div>
+     
+      <div class="menu" id="menu">
+        <a class="menu-icon" href="#menu">
+            <img src="./assets/images/Enabled.svg" alt="" srcset="">
+          </a>
```
Comment (iambenkis) [inline index.html]: - [ ] Kindly wrap the hamurger icon or the menu image inside a `nav` tag , as requested in the requieremnts 👍 

```bash
<nav>
      <a>...</a>
       ...
</nav>
```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portofolio</title>
+</head>
+<body>
+    <header class="header">
+      <div class="logo">My Logo</div>
+
```
Comment (iambenkis) [inline index.html]: - [ ] For semantic reason it could be better if you use `a` tag for the logo instead of a `div` 😄

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+  height: 718px;
+  width: 100%;
+  background-color: #fff;
+  background-image: url('./assets/images/header-bg.svg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 150px 24px;
+  background-origin: 0 0;
+  border-bottom-left-radius: 20%;
+}
```
Comment (iambenkis) [inline styles.css]: - [ ] Good job adding the `bio section` to your web page 👏. The requirement says that you should stick as much as possible to the Figma design , I noticed that the background position is not exactly as it is in the Figma design. Kindly fix that 👍 

| Your design | Figma design |
| :---: | :---: |
|  Capture | Capture3 |

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @habitoz   
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+  height: 718px;
+  width: 100%;
+  background-color: #fff;
+  background-image: url('./assets/images/header-bg.svg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 150px 24px;
+  background-origin: border-box;
+  border-bottom-left-radius: 20%;
+}
```
Comment (ichala) [inline styles.css]: - [ ] as mentioned by the previous reviewer the background position still doesn't match the Figma design , kindly fix it by updating your code to the snippet below : 

``` CSS
.bio {
    height: 718px;
    width: 100%;
    background-color: #fff;
    background-image: url(./assets/images/header-bg.svg);
    background-repeat: no-repeat;
    /* set background position to center */
    background-position: center;
    /* change background size to contain*/
    background-size: contain;
    padding: 150px 24px;
    background-origin: border-box;
    border-bottom-left-radius: 20%;
}
```

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @habitoz, 




Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡
 
✅ Descriptive PR 👍 
✅ Good commit messages 👍 
✅ Linters are passing 👏 
✅ Nice design :heavy_check_mark: 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**